test/tab.test.ts

```diff
@@ -164,13 +186,35 @@ describe('A tab', () => {
         }));
     });
 
+    test('should emit the "urlChange" event with a sanitized URL on history push state', () => {
+        const sanitizedUrl = 'sanitized://example';
+
+        const sanitizer: UrlSanitizer = jest.fn().mockReturnValue(new URL(sanitizedUrl));
+        const tab = new Tab(tabId, true, sanitizer);
+        const listener = jest.fn();
+
+        tab.addListener('urlChange', listener);
+
+        window.history.pushState({}, 'Other page', '/other-page');
+
+        expect(sanitizer).toHaveBeenCalledWith('http://localhost/other-page');
```

Comment:
Is this actually passing? The code is not calling the function, I don't see where the sanitizer would be called from the event listener

Reply:
Ok, found it

Reply:
Yes, I just [moved it to a higher level](https://github.com/croct-tech/sdk-js/pull/102/files#diff-e175ebe06b3b78c717703f4bd512d3ac3db2feac141e75571ed6e3e41b6f1279R67)